packages/abstractions/kiota_abstractions/request_response_logger.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC, abstractmethod
+from typing import Any, Dict
+
+class RequestResponseLogger(ABC):
```

Comment:
could we look into an alternative that does not expand the public API surface here please?